packages/main/src/components/AnalyticalTable/columnHeader/ColumnHeaderModal.tsx

```diff
@@ -13,6 +13,7 @@ import { StandardListItem } from '../../../lib/StandardListItem';
 import { Icon } from '../../../webComponents/Icon';
 import { FlexBox, FlexBoxAlignItems } from '../../FlexBox';
```

Comment:
Not your change, but  just saw it
```suggestion
import { FlexBox } from '../../lib/FlexBox';
import { FlexBoxAlignItems } from '../../lib/FlexBoxAlignItems';
```